config/daemon/index.md

```diff
@@ -220,15 +220,28 @@ The daemon logs may help you diagnose problems. The logs may be saved in one of
 a few locations, depending on the operating system configuration and the logging
 subsystem used:
 
-| Operating system      | Location                                                                                 |
-|:----------------------|:-----------------------------------------------------------------------------------------|
-| RHEL, Oracle Linux    | `/var/log/messages`                                                                      |
-| Debian                | `/var/log/daemon.log`                                                                    |
-| Ubuntu 16.04+, CentOS | Use the command `journalctl -u docker.service` or `/var/log/syslog`                      |
-| Ubuntu 14.10-         | `/var/log/upstart/docker.log`                                                            |
-| macOS (Docker 18.01+) | `~/Library/Containers/com.docker.docker/Data/vms/0/console-ring`                         |
-| macOS (Docker <18.01) | `~/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring` |
-| Windows               | `AppData\Local`                                                                          |
+| Operating system          | Location                                                                                                                                 |
+|:--------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------|
+| Linux                     | Use the command `journalctl -xu docker.service` (or read `/var/log/syslog` or `/var/log/messages`, depending on your Linux Distribution) |
+| macOS (`dockerd` logs)    | `~/Library/Containers/com.docker.docker/Data/log/vm/dockerd.log`                                                                         |
+| macOS (`containerd` logs) | `~/Library/Containers/com.docker.docker/Data/log/vm/containerd.log`                                                                      |
+| Windows                   | `AppData\Local`                                                                                                                          |
```

Comment:
Wasn't sure if this location is still correct; probably needs two separate mentions (WSL2 + Native Windows containers)

@djs55 @StefanScherer

Reply:
@thaJeztah for WSL 2 it is `AppData\Roaming\Docker\log\vm\dockerd.log` and `AppData\Roaming\Docker\log\vm\containerd.log`
Windows container mode uses the Event Log AFAIK.

Reply:
Thanks; I'll add those